includes/class-monk-links.php

```diff
@@ -0,0 +1,523 @@
+<?php
```

Comment:
Review all code standards.
- Remove `@link` and `@author` tags, we can receive a lot of spam.
- Pay attention on white-spaces used in all methods.
- Keep consistent with other files.
- No blank lines before and after declaring a method.